src/waveform/waveformwidgetfactory.cpp

```diff
@@ -673,6 +662,7 @@ void WaveformWidgetFactory::render() {
         // WSpinnys are also double-buffered QGLWidgets, like all the waveform
         // renderers. Render all the WSpinny widgets now.
         emit renderSpinnies(m_vsyncThread);
+        emit renderVuMeters(m_vsyncThread);
```

Comment:
`m_vsyncThread` is not used so no need to pass it, no?
```suggestion
        emit renderVuMeters();
```

Reply:
Sure. I want to use timer information from the vsyncThread instead of having a timer (member m_timer) in each WVuMeter, but until I do so I will either remove the argument or leave a TODO.

Reply:
I will try to use the timer info.

Reply:
done. using the existing vsyncthread timer instead of a timer per vumeter